Loggers/network.py

```diff
@@ -89,7 +94,9 @@ def __init__(self, simplemonitor, port, key=None):
         if key is None or key == "":
             raise util.LoggerConfigurationError("Network logger key is missing")
         Thread.__init__(self)
-        self.sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+        # On Linux, if /proc/sys/net/ipv6/bindv6only=0 (the default), then
```

Comment:
My FreeBSD system has `net.inet6.ip6.v6only: 1` as default, which means this behaviour isn't portable:

```
~src % cat test-socket.py
import socket

sock = socket.socket(socket.AF_INET6, socket.SOCK_STREAM)
sock.bind(('', 31337))
while True:
    sock.listen(10)

~src % python3 test-socket.py &
[1] 24118
~src % sockstat -l | grep 31337
james    python3.6  24118 3  tcp6   *:31337               *:*
```

May be prudent (especially as this is an option configurable on individual systems) to make the address family a config option?